org.openmw.OpenMW.json

```diff
@@ -0,0 +1,265 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.gnome.Platform",
```

Comment:
If this uses Qt you probably don't need anything from this? Just use `org.freedesktop.Platform` if so.